components/Match/renderMatch.js

```diff
@@ -0,0 +1,324 @@
+/* eslint-disable */
+import { isRadiant } from '../../utility';
```

Comment:
Similar to the other file, I think we should make an issue for upgrading this so we don't lose track.